compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -2364,6 +2364,15 @@ static LogicalResult setRootConfig(IREE::GPU::TargetAttr target,
   if (succeeded(setVectorDistributionConfig(target, entryPointFn, computeOp))) {
     return success();
   }
+  // TODO : remove this when tile and fuse backend config becomes the default
+  // for matmul.
+  if (!clGPUEarlyTileAndFuseMatmul && isROCmBackend(target)) {
```

Comment:
```suggestion
  if (!clGPUEarlyTileAndFuseMatmul) {
```
We shouldn't need the rocm check here.

Reply:
actually we shouldn't need to check the flag either, can we just always have this on?